Package.swift

```diff
@@ -854,7 +856,8 @@ if ProcessInfo.processInfo.environment["SWIFTCI_USE_LOCAL_DEPS"] == nil {
         .package(url: "https://github.com/swiftlang/swift-syntax.git", branch: relatedDependenciesBranch),
         .package(url: "https://github.com/apple/swift-system.git", "1.1.1" ..< "1.4.0"),
         .package(url: "https://github.com/apple/swift-collections.git", "1.0.1" ..< "1.2.0"),
-        .package(url: "https://github.com/apple/swift-certificates.git", "1.0.1" ..< "1.4.0"),
+        .package(url: "https://github.com/apple/swift-certificates.git", "1.0.1" ..< "1.6.0"),
```

Comment:
Needed to pull in https://github.com/apple/swift-certificates/pull/180 which fixes a Windows build break